cl/recap/management/commands/pacer_iquery_scraper.py

```diff
@@ -0,0 +1,77 @@
+# coding=utf-8
+from __future__ import print_function
+
+import json
+import re
+
+import requests
+from django.conf import settings
+
+from cl.celery import app
+from cl.lib.command_utils import VerboseCommand
+from cl.lib.pacer import map_cl_to_pacer_id
+from cl.lib.pacer_session import get_or_cache_pacer_cookies
+from cl.recap.mergers import (
+    update_docket_metadata,
+    add_bankruptcy_data_to_docket,
+)
+from cl.search.models import Docket
+from cl.search.tasks import add_items_to_solr
+from juriscraper.pacer import CaseQuery, PacerSession
+
+
+def get_dockets():
+    visits = requests.get(
+        settings.MATOMO_URL,
+        timeout=1,
+        params={
+            "idsite": settings.MATOMO_SITE_ID,
+            "module": "API",
+            "method": "Live.getLastVisitsDetails",
+            "period": "week",
+            "format": "json",
+            "date": "today",
+        },
+    )
+    visitsjson = json.loads(visits.text)
+    for item in visitsjson[:10]:
+        print(item["actionDetails"][0]["url"])
+    urllist = [
+        item["actionDetails"][0]["url"]
+        if "url" in item["actionDetails"][0]
+        else ""
+        for item in visitsjson
+    ]
+    urllistasstring = "|".join(urllist)
```

Comment:
This is a bit of a funny way to do this. You have the URLs in a list, why not just loop over the list?

Reply:
I wanted to return a clean list of docket IDs. We could loop over the list but then it couldn't be in a separate function. Maybe that's fine. It's not that long to all be in one function.

Reply:
Also I want to dedup the dockets, and I can't do that as URLs, since some will have &page=2 or other stuff. I need to get to the docket IDs before I can dedup, I think?

Reply:
I suppose I could loop over the list and append to a set. Would that be faster than combining into one string and regex like I did?